server/src/test/java/org/apache/druid/server/emitter/EmitterModuleTest.java

```diff
@@ -87,6 +108,78 @@ public void testInvalidEmitterType()
     makeInjectorWithProperties(props).getInstance(Emitter.class);
   }
 
+  @Test
+  public void testEmitterForTaskContainsAllTaskDimensions()
+  {
+    Properties props = new Properties();
+    props.setProperty("druid.emitter", "stub");
+    EmitterModule emitterModule = new EmitterModule();
+    emitterModule.setProps(props);
+
+    ImmutableSet<NodeRole> nodeRoles = ImmutableSet.of();
+
+    TestTaskHolder testTaskHolder = new TestTaskHolder("d", "e", "a", "w");
```

Comment:
Maybe use strings that are easier to follow, e.g. 
`datasource = wiki`, `taskID = id1`, `taskType = type1`, `groupId = group1`, etc.